gulpfile.js

```diff
@@ -85,14 +85,14 @@ gulp.task('copy-app-common', function() {
     gulp.src('./node_modules/whatwg-fetch/fetch.js')
       .pipe(rename('js/polyfills/fetch.js'))
   )
-  .pipe(gulp.dest(DIST_APP_ROOT));
+    .pipe(gulp.dest(DIST_APP_ROOT));
```

Comment:
__question:__ Is it by accident or you think this alignment is better (here and in several places below)? Anyway up to you, I don't really care :)